src/SelfService/Infrastructure/Api/MembershipApplications/MembershipApplicationController.cs

```diff
@@ -229,4 +229,25 @@ public async Task<IActionResult> SubmitApproval(string id)
             );
         }
     }
+
+    [HttpGet("me")]
+    [ProducesResponseType(typeof(MembershipApplicationListApiResource), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status401Unauthorized, "application/problem+json")]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound, "application/problem+json")]
+    public async Task<IActionResult> MembershipsThatUserCanApprove()
```

Comment:
Should we not call this function 'Me'?

That would allow us to expand it in the future, if we need that.
In any case I think there should be some correlation between URL and function name 🤷